app/templates/search/agenda-items.hbs

```diff
@@ -1,57 +1,79 @@
-              {{sortType.label}}
-            </option>
-          {{/each}}
-        </select>
-      </div>
-      <div class="auk-u-mb-2">
-        <Auk::Label>{{t "type"}}</Auk::Label>
-        <Auk::CheckboxList>
-          <AuControlCheckbox
-            @label={{t "note"}}
-            @checked={{this.includeNotas}}
-            @onChange={{this.toggleIncludeNotas}}
+{{page-title (t "agendaitems")}}
+<AuMainContainer as |m|>
+  <m.sidebar>
+    <div class="au-c-sidebar">
+      <div class="au-c-sidebar__content">
+        <Search::Navigation />
+        <div class="au-o-box au-o-box--small au-o-flow au-o-flow--tiny">
+          <Search::DateRangeFilter 
+            @onDateFromChange={{this.onDateFromChange}}
+            @onDateToChange={{this.onDateToChange}}
+            @onDateToClear={{set this "dateTo" undefined}}
+            @onDateFromClear={{set this "dateFrom" undefined}}
+            @dateFrom={{this.dateFrom}}
+            @dateTo={{this.dateTo}}
           />
-          <AuControlCheckbox
-            @label={{capitalize (t "announcement")}}
-            @checked={{this.includeMededelingen}}
-            @onChange={{this.toggleIncludeMededelingen}}
-          />
-        </Auk::CheckboxList>
-      </div>
-      <div>
-        <Auk::Label>{{t "version"}}</Auk::Label>
-        <AuControlCheckbox
-          @label={{t "only-on-final-agenda"}}
-          @checked={{this.latestOnly}}
-          @onChange={{this.toggleLatestOnly}}
-        />
+          <Search::MinisterFilter />
+          <div class="au-c-card au-o-box au-o-box--small">
+            <Auk::Label>{{t "type"}}</Auk::Label>
+            <Auk::Checkbox
+              @checked={{this.includeNotas}}
+              @label={{t "note"}}
+              {{on "input" this.toggleIncludeNotas}}
+            />
+            <Auk::Checkbox
+              @checked={{this.includeMededelingen}}
+              @label={{capitalize (t "announcement")}}
+              {{on "input" this.toggleIncludeMededelingen}}
+            />
+          </div>
+          <div class="au-c-card au-o-box au-o-box--small">
+            <Auk::Label>Agenda versie</Auk::Label>
+            <Auk::Checkbox
+              @checked={{this.latestOnly}}
+              @label={{t "only-on-final-agenda"}}
+              {{on "input" this.toggleLatestOnly}}
+            />
+          </div>
+        </div>
       </div>
     </div>
-  </div>
-  <div class="auk-o-grid-col-10">
-    <div>
+  </m.sidebar>
+  <m.content @scroll={{true}}>
+    <div class="au-o-box au-o-box--small">
       {{#if this.model.length}}
+        <div
+          class="au-u-flex au-u-flex--no-wrap au-u-flex--spaced au-u-flex--between au-u-flex--vertical-center au-u-margin-bottom-small"
+        >
+          <p class="au-u-muted">
+            <strong class="au-u-medium">1-{{this.size}}</strong>
+            resultaten voor
+            <strong class="au-u-medium">"{{this.searchTextBuffer}}"</strong>
```

Comment:
`this.searchTextBuffer` doesn't exist on the agenda-items controller. It's a query param from the base `search` route, but you could just set in on the agenda-items controller as a tracked property.

```diff
//app/routes/search/agenda-items.js
  setupController(controller) {
    super.setupController(...arguments);
    controller.emptySearch = isEmpty(this.paramsFor('search').searchText);
+   controller.searchTextBuffer = this.paramsFor('search').searchText;

    if (controller.page !== this.lastParams.committed.page) {
      controller.page = this.lastParams.committed.page;
    }
  }

//app/controlers/search/agenda-items.js
  @tracked emptySearch;
+ @tracked searchTextBuffer;
```

Reply:
Same applies for the search/cases routes

Reply:
I'm not even sure how this got here, I don't think it needs to be there